tests/integration/kubernetes/run_kubernetes_nv_tests.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# Copyright (c) 2025 NVIDIA Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+kubernetes_dir=$(dirname "$(readlink -f "$0")")
+source "${kubernetes_dir}/../../common.bash"
```

Comment:
Can you add `# shellcheck disable=SC1091 # import based on variable` to stop the shellcheck warning please